task/git-clone-oci-ta/0.1/git-clone-oci-ta.yaml

```diff
@@ -151,6 +146,8 @@ spec:
       - name: trusted-ca
         mountPath: /mnt/trusted-ca
         readOnly: true
+      - name: source
+        mountPath: /var/source
```

Comment:
That won't work because `CHECKOUT_DIR` is not a parameter. It's an environment variable. I don't see a way of using that data [here](https://tekton.dev/docs/pipelines/variables/). We could make CHECKOUT_DIR a parameter and then simply use that in both places. But turning this into a parameter, makes it user visible which is not really the intention. Let's leave it as is.